src/constants.ts

```diff
@@ -1,3 +1,7 @@
+export const CURRENT_SCORE_VERSION = 1.2; // Increase this every time we add a new source
+
+export const ATTESTER_PUBLIC_KEY = '0xd8dA8766431366E9aD63eA6D16d9199cece2aA08';
```

Comment:
Small change, but this is technically an address (a hash of the public key) and not the public key itself - might be worth to change to be exact.